app/controllers/application_controller.rb

```diff
@@ -39,6 +39,19 @@ def verify_admin
     (current_user.nil?) ? redirect_to(root_path) : (redirect_to(root_path) unless current_user.is_admin?)
   end
 
+  def tracker(ga_category, ga_action, ga_label, ga_value=nil)
```

Comment:
Sorry for the confusion - you're right that not passing the correct number of arguments leads to `ArgumentError: wrong number of arguments` unless you set the default. I guess what I mean was there's no need to combine passing `nil` as a parameter when calling `tracker()` _as well as_ defaulting the param to `nil` as you're doing here; it seems superfluous. No harm done though, and the PR gets my 👍  with or without this! :)